### ISSUE TITLE
Calculate and show pipeline in calendar time

### ISSUE THREAD

**Issue by bicschneider** [eficode/pipeline-the-game#263]
As a user I would like to see the total "cost" of time in the game board by calculating longest path throught the pipeline

**Comment by sofusalbertsen** [comment]
Yes, during development, when asked about what would be plausible to type there people wrote all kinds of things like "4 minutes more or less" Or "it depends", so we can't rely that people only type metrics that are computer parseable in.
Do you have a suggestion for how to handle that?
We hit the same problem in #258
